std_msgs/Header header
auto_parking/ParkingSlot[] slots
